### DISCUSSION TITLE
Coding with Obsidian's native PDF viewer or PDF++ plugin

### DISCUSSION THREAD

**Discussion by Epicstrangerr** [chrisgrieser/obsidian-quadro#8] [Ideas]
Most of my data in one research is in PDF. It could be useful for Quadro be able to code in the native pdf viewer or extract highlights in the PDF ++ plugin. This would also be helpful for doing literature reviews.

**Comment by chrisgrieser** [comment]
Quadro has been designed to work with markdown files, and as such, working with PDFs would require an entirely new plugin which is out of scope for this project.

What you can do is to extract the text from a PDF into markdown files and then work with the text (googling finds you various tools for that). This has also various other advantages (smaller file sizes, ability to use many other tools, etc.) and is generally what I would recommend doing.
